src/System.Runtime/tests/System/TimeZoneInfo.cs

```diff
@@ -96,7 +99,7 @@ public static void ValidateLibyaTimeZoneTest()
     [ActiveIssue(846, PlatformID.AnyUnix)]
```

Comment:
There are still 8 tests marked with 846 attributes.  I've logged https://github.com/dotnet/corefx/issues/2465 to handle parsing the tzfile correctly, which should take care of most of them.  As I find other issues, I will log them as I go.